trollflow2/plugins/__init__.py

```diff
@@ -216,6 +217,23 @@ def prepared_filename(fmat, renames):
         yield orig_filename
 
 
+def format_decoration(fmat,  fmat_config):
+    """
+    Format decoration in fmt_config according to fmat.
+    """
+    fmat_config_local = copy.deepcopy(fmat_config)
+    if "decorate" in fmat_config:
+        for deco in fmat_config_local["decorate"]["decorate"]:
+            if "text" in deco and "txt" in deco["text"]:
+                try:
+                    deco["text"]["txt"] = deco["text"]["txt"].format(**fmat)
+                except KeyError:
+                    LOG.debug('Could not formate: %s.', str(deco["text"]["txt"]))
+        return fmat_config_local
+    else:
+        return fmat_config_local
```

Comment:
```suggestion
    return fmat_config_local
```
The `else` branch is redundant.